### ISSUE TITLE
Is it recommended to use sidecar's log output information in your own agent code?

### ISSUE THREAD

**Issue by binsee** [huan/sidecar#27]
My agent code needs to output log information, but the information directly output by `console.log()` is confusing and not conducive to reading.
My code uses Brolog to output formatted logs, and the sidecar also uses Brolog. 

Moreover, the sidecar also encapsulates the log in the agent, which can output the log information in the agent to Brolog.

````js
log.verbose(
  'SidecarAgent',
  'Interceptor.attach(%s) onEnter()',
  'testCallback',
)
````

So I thought of reusing the log output function of the sidecar itself, so that the log information can be output to Brolog.

E.g:

```js
// init-agent-script.js
const LogPre = 'Demo-Agent'
const logger = {
  logLevel: {
    info: 0,
    verbose: 1,
    silly: 2,
  },
  level: (newLevel) => log.level(newLevel),
  verbose: (...args) => log.verbose(LogPre, ...args),
  silly: (...args) => log.silly(LogPre, ...args),
}

const BaseAddr = __sidecar__moduleBaseAddress
const testOffset = 0x10000

Interceptor.attach(BaseAddr.add(testOffset), {
  onEnter: function (args) {
    logger.verbose('Test hook arg: %d', args[0])
  },
})
```

In this way, the log information can be effectively classified and filtered according to the prefix. For example, the following code is used in ts:

```typescript
// index.ts
import { log } from 'brolog'
const PRE = 'Test-Demo'

log.prefix(/demo/i)

// ...
```

I think that reusing the log output function of sidecar can reduce the time to write an additional set of log output functions, which can greatly facilitate developers.

In addition, in the agent, sidecar can only output logs of verbose and silly levels, hoping to support all levels of log output of Brolog.
- [ ] `ERR`  (`Not currently supported`)
- [ ] `WARN`  (`Not currently supported`)
- [ ] `INFO`  (`Not currently supported`)
- [x] `VERB`
- [x] `SILL`

**Comment by huan** [comment]
Yes, it is recommended to use the internal `log` as a `Logger` for logging messages.

It is designed to bridge the log message from the Frida context to the Node.js context so that we can get a consistent logger interface (and level control) in the Node.js program.

My suggestion would be to use it as possible as we can in our agent code.

For your log level suggestion, I think it's good. The reason that the current code only supports `verb` and `sill` is that when I write the Sidecar there's too much work, so I just implemented those two.

I'd like to accept a PR to improve it.
